game_eggs/steamcmd_servers/dont_starve/egg-don-t-starve-together.json

```diff
@@ -1,24 +1,22 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1",
+        "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2021-07-02T04:29:47+03:00",
+    "exported_at": "2023-01-07T09:58:40-05:00",
     "name": "Don't Starve Together",
     "author": "parker@parkervcp.com",
     "description": "Don\u2019t Starve Together is an uncompromising wilderness survival game full of science and magic.",
-    "features": [
-        "steam_disk_space"
-    ],
-    "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_source"
-    ],
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/games:source": "ghcr.io\/parkervcp\/games:source"
```

Comment:
Should this not be the steamcmd image now? The source was still there for compatibility

Reply:
I just went from source to source